web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
Ok I can fix that for both the locales, and actually im looking at the code in web.go and the route for /docs is 'api.AppHandlerIndependent' and to use the **user locale** it should be `api.UserRequired` so I can also change that